server/public/plugin/api.go

```diff
@@ -1306,6 +1306,12 @@ type API interface {
 	// @tag User
 	// Minimum server version: 9.8
 	UpdateUserRoles(userID, newRoles string) (*model.User, *model.AppError)
+
+	// GetPluginId returns the plugin's ID.
+	//
+	// @tag ID
+	// Minimum server version: 9.12
```

Comment:
There is no 9.12 release. The next release where this change can be included is 10.1
```suggestion
	// Minimum server version: 10.1
```